app/templates/pages/justice-and-other-user.html

```diff
@@ -13,26 +13,28 @@
 {% endblock %}
 
 {% block content %}
-<h1 class="govuk-heading-l">GitHub Organisation Invitation</h1>
+<h1 class="govuk-heading-l">Verify your work email address</h1>
 
 <p class="govuk-body">
   To access the Ministry of Justice GitHub organisations you've selected, your email address <strong>{{email}}</strong>
```

Comment:
This section needs to be replaced by "This secure step is essential to confirm your identity and initiate the process of joining our GitHub organisations."